naming/src/main/java/com/alibaba/nacos/naming/consistency/persistent/raft/RaftStore.java

```diff
@@ -139,7 +141,7 @@ public synchronized Datum load(String key) throws Exception {
     }
     
     private boolean isDatumCacheFile(String fileName) {
-        return fileName.endsWith(CACHE_FILE_SUFFIX);
+        return fileName.endsWith(RAFT_CACHE_FILE_SUFFIX) || fileName.startsWith(RAFT_CACHE_FILE_PREFIX);
```

Comment:
I think the second condition maybe too wide. should be add `.naming` for it?

Reply:
Okay